src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionTest.java

```diff
@@ -244,7 +266,7 @@ void listConsumerSubscriptions(VertxTestContext context) throws InterruptedExcep
                 .send(ar -> {
                     context.verify(() -> {
                         assertThat(ar.succeeded(), is(true));
-
+                        listSubscriptions.complete(true);
```

Comment:
you are just skipping all the assertions here I guess because the `listSubscriptions.get` is waiting and when it's completed there is no time for executing the assertions and the code after the get is executed and the test exits.